docs/docs/APIdocs/spark.ml/DLClassifier.md

```diff
@@ -0,0 +1,35 @@
+## DLClassifier ##
+
+**Scala:**
+```scala
+val classifier = new DLClassifier[Float](
```

Comment:
In other API's we recommend user to use API's without [T] and import NumericFloat.